libbeat/dashboards/import_dashboards.go

```diff
@@ -677,11 +676,13 @@ func main() {
 	if importer.cl.opt.Dir != "" {
 		if err = importer.ImportKibana(importer.cl.opt.Dir); err != nil {
 			fmt.Println(err)
+			os.Exit(1)
```

Comment:
Throughout the main method where error logging is performed I think this should go to stderr (IIRC `fmt.Fprintln(os.stderr, ...)`).